indico/modules/events/sessions/templates/management/sessions_list.html

```diff
@@ -0,0 +1,64 @@
+{% extends 'layout/full_content_base.html' %}
+
+{% from 'events/sessions/management/_sessions_list.html' import render_sessions_list %}
+
+{% block title %}
+    {%- trans %}Sessions{% endtrans -%}
+{% endblock %}
+
+{% block content %}
+    <div class="sessions">
+        <div class="toolbar">
+            <div class="group left">
+                <a class="icon-checkbox-checked i-button arrow left icon-only js-dropdown"
+                   href="#" title="{% trans %}Select{% endtrans %}" data-toggle="dropdown"></a>
+                <ul class="dropdown">
+                    <li><a href="#" id="select-all">{% trans %}All{% endtrans %}</a></li>
```

Comment:
You may want to take a look into [this](https://github.com/OmeGak/indico/commit/be8b60f6f7b1d0a0faabdc8db7e0f34e04b5ca9e). It should be getting merged soon.